core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -326,7 +326,7 @@
   MASTER_WALOG_CLOSER_IMPLEMETATION("master.walog.closer.implementation",
       "org.apache.accumulo.server.master.recovery.HadoopLogCloser", PropertyType.CLASSNAME,
       "A class that implements a mechansim to steal write access to a file"),
-  MASTER_FATE_METRICS_ENABLED("master.fate.metrics.enabled", "false", PropertyType.BOOLEAN,
+  MASTER_FATE_METRICS_ENABLED("master.fate.metrics.enabled", "true", PropertyType.BOOLEAN,
```

Comment:
Should these be enabled by default when they weren't before?

Reply:
They should be disabled by default - my mistake for not changing back from testing - will restore default before merge.